internal/model/game.go

```diff
@@ -1 +1,20 @@
 package model
+
+type GameScore struct {
+	BlueScore  int `json:"blueScore"`
+	WhiteScore int `json:"whiteScore"`
+}
```

Comment:
Is there a sense to keeping a public struct? Maybe better would be to add a method that will return the pointer to the private method? 

```suggestion
type gameScore struct {
	BlueScore  int `json:"blueScore"`
	WhiteScore int `json:"whiteScore"`
}

func New() *gameScore {
	return &gameScore{}
}
```

It prevents misunderstanding related to importing packages when a user would not know if he should create a struct manually or use a specific function